### PR TITLE
Added a script to install the dependencies

### PR DESCRIPTION
**Pull request by viragtripathi**
Added a simple script to install the dependencies to avoid the manual steps which can lead to errors.

### CODE REVIEW

**Comment by bmansheim** [comment]
@K-Jo We can close this because in 6.0.12 there is no need to install dependencies, correct?

**Comment by bmansheim** [comment]
@K-Jo Looks like there will be dependencies until 6.0.16. Should we merge this until then or just wait?

File: content/modules/redisgears/installing-redisgears.md [hunk 1]
```diff
@@ -20,20 +20,23 @@ On each node in the Redis Enterprise cluster:
...
-    # chown -R $osuser $modulesdatadir/rg
-    ```
-
-{{< note >}}
-- `<version>` - The version number in the format `x.y.z`.
-- `<version_integer>` - The version number in integer format `xxyyzz` or you can calculate it as `10000*x + 100*y + z`.
-- You must also run these commands on new nodes before you join them to an existing cluster.
-{{< /note >}}
+1. Save and Run this script as root:
+cat install_rg_dependencies.sh
```
Comment (K-Jo) [inline content/modules/redisgears/installing-redisgears.md:25]: This should be a code snippet

**Review by K-Jo — CHANGES_REQUESTED** [review]

**Comment by bmansheim** [comment]
This behavior will change in 6.0.12.

**Pull request closed without merging**